toolchest/rpm/utils.py

```diff
@@ -44,7 +44,7 @@ def splitFilename(nvr):
     arches = ['noarch', 'x86_64', 'i386', 'i486', 'i586',
               'i686', 'ppc', 'ppc64', 'ppc64le', 's390',
               's390x', 'aarch64', 'src']
-    arch_rx = '\.(' + '|'.join(arches) + ')$'
+    arch_rx = r'\.(' + '|'.join(arches) + ')$'
```

Comment:
Just curious here, as I suspect it will be hard to search for - what is the significance of adding the 'r' to these regexes?

Reply:
This is an interesting case, the issue is '\.' is not properly escaped based on W605 marking them as raw string (adding the 'r') fixes this issue since the string is not checked for escape characters. You will see raw strings used with regular expressions often.

Reply:
Cool, thanks for the explanation